### ISSUE TITLE
Mantener URL iguales donde sea posible

### ISSUE THREAD

**Issue by jpadilla** [Code4PuertoRico/suministrospr#43]
Reference: https://suministrospr.com/sectores/barriada-ferran-sector-san-francisco/

**Comment by tommytomtj** [comment]
What needs to be done here? I see that some urls have numbers like this one: https://suministrospr.com/sectores/342/ for Baldorioty. Is that what you mean?

**Comment by jpadilla** [comment]
@tommytomtj our current urls look like this:

```
/
/suministros/add/
/suministros/municipios/<slug:municipality>/
/suministros/<slug:slug>/
/suministros/<slug:slug>/edit/
```

We should try to keep as many URLs as we can working where it makes sense:

Example:

```
https://suministrospr.com/municipios/ponce/ -> /municipios/<slug:municipality>/
https://suministrospr.com/sectores/barriada-ferran-sector-san-francisco/ -> /suministros/<slug:slug>/
```

**Comment by tommytomtj** [comment]
Ok I got it. Ive made that change locally. Testing now.

**Comment by tommytomtj** [comment]
Is there a branch I can push to?

**Comment by miguelrios** [comment]
your own branch should be ok

**Comment by jpadilla** [comment]
@tommytomtj You should probably fork the repository and then open a pull request.

https://guides.github.com/activities/forking/

**Comment by tommytomtj** [comment]
Okay cool thanks.

**Comment by jpadilla** [comment]
Fixed via #50
